guides/source/initialization.md

```diff
@@ -76,20 +76,27 @@ A standard Rails application depends on several gems, specifically:
 * activesupport
 * actionmailbox
 * actiontext
-* arel
+* bootsnap
 * builder
 * bundler
 * erubi
+* importmap-rails
 * i18n
+* jbuilder
 * mail
 * mime-types
+* puma
 * rack
 * rack-test
 * rails
 * railties
 * rake
+* redis
```

Comment:
Although `gem "redis"` is in the Gemfile, I believe it is commented out by default, no?

Reply:
When generating a new application it does get added by default